docs/resources/routing_email_domain.md

```diff
@@ -20,8 +20,8 @@ The following Genesys Cloud APIs are used by this resource. Ensure your OAuth Cl
 ## Example Usage
 
 ```terraform
-resource "genesyscloud_routing_email_domain" "test-example-com" {
-  domain_id             = "test.example.com"
+resource "genesyscloud_routing_email_domain" "example-example-com" {
+  domain_id             = "example.example.com"
```

Comment:
@dginty4  `example.domain.com` might look better here, I think. Same for the resource ID on the line above.